modules/serverless-rn-1-26-0.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies
+//
+// * /serverless/serverless-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="serverless-rn-1-26_{context}"]
+= Release notes for Red Hat {ServerlessProductName} 1.26
+
+{ServerlessProductName} 1.26 is now available. New features, changes, and known issues that pertain to {ServerlessProductName} on {product-title} are included in this topic.
+
+[id="new-features-1.26_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 1.5.
+* {ServerlessProductName} now uses Knative Eventing 1.5.
+* {ServerlessProductName} now uses Kourier 1.5.
+* {ServerlessProductName} now uses Knative (`kn`) CLI 1.5.
+* {ServerlessProductName} now uses Knative Kafka 1.5.
+* {ServerlessProductName} now uses Knative Operator 1.3. FIXME please confirm this needs explicit listing here
+* The `kn func` CLI plug-in now uses `func` 1.8.1.
+
+* Persistent volume claims (PVCs) are now GA. PVCs provide permanent data storage for your Knative services.
+
+* The new trigger filters feature is now available as a Technology Preview (TP). It allows users to specify a set of filter expressions, where each expression evaluates to either true or false for each event.
++
+To enable new trigger filters, add the `new-trigger-filters: enabled` entry in the section of the `KnativeEventing` type in the operator config map:
++
+[source,yaml]
+----
+apiVersion: operator.knative.dev/v1alpha1
+kind: KnativeEventing
+...
+...
+spec:
+  config:
+    features:
+      new-trigger-filters: enabled
+    logging:
+...
+----
+
+* FIXME Please ACK my decision to have no changes in the release notes and GA/TP support table and deprecation tables for these Jiras linked to the release notes Jira:
+** https://issues.redhat.com/browse/SRVCOM-2151
+*** Reasoning: no release note nor support entry in the table is necessary for removal of mentions of tech that is not supported anyway (buildpacks).
+** https://issues.redhat.com/browse/SRVCOM-2152
+*** Reasoning: the revival of Functions developer preview docs has no bearing on the official docs, since developer preview is below the minimal level of support.
+** https://issues.redhat.com/browse/SRVCOM-2093
+*** Reasoning: Martin Gencur's comment suggesting to put this Jira on hold.
+
+[id="fixed-issues-1.26_{context}"]
+== Fixed issues
```

Comment:
Previously, we had this issue in known issues: 
```
The Federal Information Processing Standards (FIPS) mode is disabled for Kafka broker, Kafka source, and Kafka sink.
```
This is now fixed and probably would be good to mention it.